spec/models/comment_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe "validations" do
+    let!(:user) {create(:user)}
+    let(:worker) {user.worker}
+    let!(:ticket) {create(:ticket)}
+    let!(:comment) {create(:comment)}
+
+    it "create a comment" do
+      expect(comment).to be_valid
+    end
+
+    it "is not valid without a message" do
+      comment.message = nil
+      expect(comment).to_not be_valid
+    end
+
+    it "is not valid without a message" do
+      comment.worker_id = nil
+      expect(comment).to_not be_valid
+    end
+
+    it "is not valid without a message" do
+      comment.ticket_id = nil
+      expect(comment).to_not be_valid
+    end
+
+    it "should have one ticket" do
+      t = Comment.reflect_on_association(:ticket)
+      expect(t.macro).to eq(:belongs_to)
+    end
+    it "should return true" do
```

Comment:
bad bad test :) 
should look like

context '#check_time" do
   context 'when time is less blablabla" do
      it 'should be false" do
        ...
      end
   end

   context 'when time is more blablabla" do
      it 'should be true" do
        ...
      end
   end
end